test/unit/models/forms/DecisionFormTest.scala

```diff
@@ -42,6 +42,11 @@ class DecisionFormTest extends ModelsBaseSpec {
     "methodExclusion"      -> Seq("exclusion")
   )
 
+  private val emptyParams =
```

Comment:
Same with the others as it can just be `params.view.mapValues(_ => Seq("")).toMap`. Thanks